private/buf/cmd/buf/command/curl/curl.go

```diff
@@ -93,6 +93,9 @@ const (
 	dataFlagShortName      = "d"
 	outputFlagName         = "output"
 	outputFlagShortName    = "o"
+
+	// Other flags
+	emitDefaultsFlagName = "emit-defaults"
```

Comment:
Maybe call it "Output flags" and move the above `outputFlag*` consts down to this section?
```suggestion

	// Output flags
	outputFlagName       = "output"
	outputFlagShortName  = "o"
	emitDefaultsFlagName = "emit-defaults"
```